maintenance/wikia/HttpsMigration/migrateWikiToFandom.php

```diff
@@ -116,8 +135,18 @@ private function getTargetDomain( $sourceDomain ) {
 			}
 			return "{$parts[1]}.{$wgFandomBaseDomain}/{$parts[0]}";
 		}
-
+		$additionalDomain = true;
```

Comment:
Changing the state of an object inside a getter method is not really a "kosher" approach.